### PR TITLE
Finish 'Scorecards' section

### PR DESCRIPTION
**Pull request by dobromirmontauk**
This PR cleans up the expectations/scorecard section and makes it actually useful.

### CODE REVIEW

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    2. You (and your team!) are always aware of the critical path to launch and are actively mitigating the risk. Dependencies on other teams are well-understood - and they understand what you expect of them.
+    3. You optimize for low-latency. You are comfortable burning engineering hours, if it makes sense, to de-risk a milestone, prove product/market fit, or hit an important customer commitment.
+    4. You deeply understand the Feature Requirements (owned by PMs) and are able to make appropriate engineering/product tradeoffs. You intelligently negotiate requirements (scope/deadlines/prioritization/tech-debt) based on engineering cost vs product value. You are the final decider on _what_ gets built _when_. 
+    5. Your launches are smooth. The product works, the support cost on you and partner teams is low, and customers are happy. You intelligently tradeoff before-launch work and after launch-work - and you actually do the after-launch cleanup!
+2. **_Ownership: you are the undisputed expert of your product_**
```
Comment (joannakrk) [inline recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md:36]: expert of? maybe on or in?

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    5. You take full responsibility for everything related to your feature - every bug, outage, or unhappy customer. No excuses. 
+3. **_Customer Obsession: your effective partnership with PM levels-up our customer focus  _**
+    1. You know who your customers are and you evangelize their problems, needs, and vocabulary within your team. All members understand the bigger picture and the role of your project/team within it.
+    2. You understand the customer problems deeply enough to hold your own in a debate. You can predict what your customers will need and you use this to build technology that grows with them.
+    3. You are an effective partner to, and proxy for, your PM. You speak the same language and have the same vision. You complement each other naturally: s/he has breadth, you have depth; s/he can focus on long-term while you handle the short-term details. You help them find low-hanging product fruit through your technical expertise.
```
Comment (joannakrk) [inline recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md:45]: I'd replace s/he with they

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    5. You escalate product requirements when they are unclear or not in the _overall _best interest of your customers/users.
+4. **_Leadership: sometimes engineering is more about people than about writing code…_**
+    1. You create positive pressure to ship. You help engineers see the _why_ so they put in maximum energy on the _what_ and the _how_. You have an intensity that is hard to describe and it energizes the people around you.
+    2. You know how to communicate perfectly: the message, the medium, the audience, the timing. Your stakeholders always know the project's status and tradeoffs chosen. Your partners/dependencies always know what you need from them and keep your requirements top-of-mind. Your leadership chain always knows the project's risks and how they can help accelerate you, if they so choose.
+    3. You keep the wheels greased. You have healthy relationships with all your partner teams, Tech Leads, dependencies, etc. Your project is never delayed because you get incorrectly blocked by external factors.
```
Comment (joannakrk) [inline recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md:51]: " you get incorrectly blocked by external factors." - what does being correctly blocked mean? Maybe just remove "incorrectly"?

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    4. You escalate when appropriate: bad technical decisions, poor work quality, inappropriate workload, unreasonable technical debt, etc.
```
Comment (joannakrk) [inline recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md:52]: I wonder if in the escalations case we should not change to "you respectfuly escalate..."?
There's a right and the wrong way of doing it, just hinting at it;)

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+5. **_Inspirational: you get your engineering team excited about the work_**
+    1. You live and breathe your product. Your enthusiasm is infectious and you spread it actively through demos, celebrations, customer stories, etc.
+    2. You are a talent magnet. People seek you out to work with you, your team, and specifically the feature you are building. You understand that talent comes in many shapes and sizes and you know how to leverage them all: 20% time, sub-delegation to another person/team, temporary allocation, or even permanent moves. In all cases, you help your recruits become part of the team.
+    3. You build a team of owners, not contractors. Your engineers care about the product they are building. Your team regularly ships ideas that came _from the team_. 
+    4. You build a [Level 4 Tribe](https://en.wikipedia.org/wiki/Tribal_Leadership): the team has fun, loves what they do, and may be friends forever. 
+    5. You don’t burn yourself or your team out. You can keep an impressive pace for years.
```
Comment (joannakrk) [inline recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md:59]: This seems really low down the list... I wonder if it should not be at least mentioned somewhere above as well, where the impressive deliveries on tight deadlines live...

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    2. You push for high-quality designs yet are realistic about business needs. You weave your feature across all the required systems in a logical way, without (permanently) hacking up any piece. 
+    3. You identify missing platforms for your feature (i.e. re-usable components) and you work with appropriate long-term owners to either build them, or to take over ownership of your short-term workaround.
+    4. You raise technical hurdles with the appropriate Tech Leads and drive towards the best end-to-end technical solution that meets the business's short-term and long-term needs. 
+    5. You take pride in high-quality engineering. You ensure your product has adequate test coverage, scales as necessary, isn’t unnecessarily expensive, is easy to debug, and will be easy to maintain for years.
+
+_Credits: thanks to … for helping put together this scorecard, and to Wade Chambers whose VP of Engineering scorecard inspired me to make this Feature Lead one!_
```
Comment (joannakrk) [inline recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md:67]: feature -> warrior?

**Review by joannakrk — COMMENTED** [review]
Hey:) took a pass - feel free to incorporate or ignore.
For the scorecards, if accepted, some comments will need to be applied across all of them - I've only put them where I saw them first.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/gardener_tl_scorecard.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+    3. You manage up effectively. You are high signal-to-noise and escalate when appropriate: bad technical decisions, poor work quality, inappropriate workload, unreasonable technical debt, etc.
+    4. You maintain healthy peer relationships with other TLs and über-TLs. You keep them aware of the issues in your system and the technical challenges on your team, as well as your plans to deal with them. You know what Feature Leads need from your system in the next year.
+    5. You navigate the changing technical tides effectively (both proactively and reactively). You realize your system will not be the best solution, forever, and that past decisions may be the wrong future direction. You don’t let ego or fear prevent you from catching the next wave: you jump in head-first, learn the implementation, technology, concepts, etc - and teach your team to do the same.
+
+Some useful resources:
+
+[https://www.slideshare.net/thekua/tech-lead-skills-for-developers?next_slideshow=1](https://www.slideshare.net/thekua/tech-lead-skills-for-developers?next_slideshow=1)
```
[the remark this answers is not in the document]
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/gardener_tl_scorecard.md:65]: Cleaned up by moving to README.md

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/twitter_ad_serving_em.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
+<h1>Twitter Ad Serving</h1>
```
[the remark this answers is not in the document]
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/twitter_ad_serving_em.md]: Yeah, this is a historical one as well.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/twitter_ad_serving_em.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+    3. Generates healthy pressure from internal motivation. Bias toward action and clearly gets excited about impact. Absorbs and redistributes pressure from organization correctly (not too much, not too little). 
+    4. Experience with rebuilding large systems (100K-1M+ LOC over 12-24 months). Can explain when a rewrite is necessary vs a refactor. Demonstrated business-value vs. engineering-need trade-off evaluation.
+    5. Intuitively understands #Winning and #IncreasingCapacityToWin. Chronically discontent and wants to move faster. Focuses on impact, business-value, and continually figuring out how to move faster/build more leverage. 
+3. Large organization leader and team player
+    1. Effectively works in large organizations with varied partners: hardware engineering, Platform engineering, legal, sales, marketing, #Goldbird peer teams. Builds strong relationships and trust across 20+ teams.
```
[the remark this answers is not in the document]
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/twitter_ad_serving_em.md:34]: Oh that was a Twitter-specific organization.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/twitter_ad_serving_em.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+    2. Customer obsession. Listens to customers and figures out how to solve their real problems. "Work vigorously to earn and keep the trust."
+    3. Chronically discontent with the organization. Owner of #Goldbird and Twitter - everything is their problem and they prioritize and tackle the most important things first. Has their head screwed-on tight: they know why they are here and what they are doing. Not passive; actively moving something forward.
```
[the remark this answers is not in the document]
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/twitter_ad_serving_em.md:36]: This is a historical scorecard that I imported from Twitter so I kind of don't want to change it.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/twitter_billing_payments_em.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+3.  Billing & payments technical/domain expert
+    1. Loves solving the technical challenges around legal/financial compliance. Can intelligently discuss how SOX, PCI or accounting needs translate into actual technological requirements.
+    2. Values accuracy, correctness, and consistency over speed, improvisation, innovation. But not too much.
+    3. Deep production experience. Has opinions on what 'healthy' systems look like and can quickly determine if a particular system qualifies. Comfortable being responsible for mission-critical systems ($XB and 24/7 team oncall). 
+    4. Strong nose for quality software and able to call out code- and system-smells quickly. Does not unnecessarily sacrifice quality and excellent judgement on hard-to-pay-off debt vs cheap technical debt. 
+    5. Big-picture thinker. Can draw the business, feature, and technical picture for a company-wide billing & payments platform. Works closely with Product Manager to draw out 3+ year strategy for the team. 
+    6. Bonus: track record of publications/blog posts, OSS, etc.
```
[the remark this answers is not in the document]
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/twitter_billing_payments_em.md:43]: Specifically billing & payments.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/uber_tl_scorecard.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+Make a copy of this [template](https://docs.google.com/spreadsheets/d/1PX61irzc6eCxgl2tzEzE6aKF92l7EoHcVac7Nb1Beko/edit#gid=0) and use the following rating scale:
+
+
+*   1 - You’re not doing this and aren’t likely to in the future.
+*   2 - You have significant areas where you need to improve.
+*   3 - You do this well.
+*   4 - You do this amazingly — keep it up.
+*   5 - OMG how do you do this so well???
+
+<h2>How your organization judges you:</h2>
```
[the remark this answers is not in the document]
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/uber_tl_scorecard.md:25]: Agreed. How could we make this scorecard not a 'architecture astronaut' that isn't practical?

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/uber_tl_scorecard.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+
+1. **_Leader: You lead by influence, not by fiat_**
+    1. You set the _direction_ on where our **_technology_** needs to end up in 2-5 years, by deeply understanding all the business constraints.
```
[the remark this answers is not in the document]
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/uber_tl_scorecard.md]: Good call out, added.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/uber_tl_scorecard.md [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+6. **_Communicator_**
+    1. You are an information router who connects people who need to talk to each other. You regularly save the organization from duplicate work and unnecessary headaches.
+    2. You represent the engineering team to the rest of the company, especially non-technical partners (sales/marketing/legal/finance/etc).
+    3. You educate VP/Director/senior EMs and PMs on the technical challenges that the organization faces. You partner with them to make the best staffing and priority decisions for the organization and company.
+    4. You **write things down**. Your documents are top-notch and stand on their own legs, no explanation/conversation required. They withstand the test of time and are updated as necessary to be the Source of Truth. 
+    5. You wield all the communication tools (Slack, email, JIRA, TDDs, etc) flawlessly for maximum leverage. You seem to be everywhere at all times; other engineers assume you are reading everything they write and improve their own communication in response.
+
+Useful resources:
+
+[https://www.slideshare.net/thekua/the-well-rounded-architect-76127395](https://www.slideshare.net/thekua/the-well-rounded-architect-76127395)
```
[the remark this answers is not in the document]
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/uber_tl_scorecard.md]: Thanks, moved to the README.md and fixed the link.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+
+<h2>How your manager judges you:</h2>
+
+
+
+
+1. **_Execution: you make commitments and you consistently hit them_**
+    1. You have a plan. It's aggressive but realistic, with unambiguous goals, and approved by the relevant experts/stakeholders. It's also flexible: you consciously update your plan as you learn or the world changes around you. You commit to specific dates in your plan, and you regularly hit them. You have a magical ability to see around corners.
+    2. You (and your team!) are always aware of the critical path to launch and are actively mitigating the risk. Dependencies on other teams are well-understood - and they understand what you expect of them.
+    3. You optimize for low-latency. You are comfortable burning engineering hours, if it makes sense, to de-risk a milestone, prove product/market fit, or hit an important customer commitment.
```
[the remark this answers is not in the document]
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md:29]: Yes... by 'low latency' I mean that from request -> delivery of the feature/bug/whatever is very fast. So low on process, tech debt is OK, etc etc. How can I clean this up?

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md [hunk 16]
```diff
@@ -0,0 +1,70 @@
...
+    4. You deeply understand the Feature Requirements (owned by PMs) and are able to make appropriate engineering/product tradeoffs. You intelligently negotiate requirements (scope/deadlines/prioritization/tech-debt) based on engineering cost vs product value. You are the final decider on _what_ gets built _when_. 
+    5. Your launches are smooth. The product works, the support cost on you and partner teams is low, and customers are happy. You intelligently tradeoff before-launch work and after launch-work - and you actually do the after-launch cleanup!
```
[the remark this answers is not in the document]
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md]: Fixed, thanks.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md (hunk 1 above)
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md]: Fixed.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md (hunk 2 above)
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md]: Fixed.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md (hunk 3 above)
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md]: I'll change it to 'unnecessarily'. Sometimes you get blocked and it's necessary, e.g. hardware doesn't show up on time b/c of a delay in the ports...

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md (hunk 4 above)
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md]: Oh I love that, fixed.

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md (hunk 5 above)
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md:55]: Nitpicking :)

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md (hunk 6 above)
Comment (dobromirmontauk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md]: Fixed!

**Review by dobromirmontauk — COMMENTED** [review]
THANK YOU for the wonderful comments. I made most of the changes.

**Comment by joannakrk** [comment]
LGTM:) Is closing of all the comments required before you can merge?

**Comment by dobromirmontauk** [comment]
> LGTM:) Is closing of all the comments required before you can merge?

I currently have it set that all conversations need to be resolved + an approval before merging. Mind sending me approval? :)

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/gardener_tl_scorecard.md [hunk 17]
```diff
@@ -0,0 +1,76 @@
...
+<h2>How your manager judges you:</h2>
+
+
+
+
+1. **_Quality: Quality you can taste<sup>®</sup>_**
+    1. You have **strong opinions** (loosely held!) about what makes good code and good design. You are comfortable saying "no" to work that doesn't meet your bar. Your codebase looks like it was written by a single person.
+    2. You are constantly improving the state of your code and system, making it easier to develop in. This includes (but is not limited to) testing, releasing, debugging tools, documentation, automation, logging, monitoring, and overall system cost in $$$.
+    3. You maintain high production quality of your system. It requires little human intervention and provides high signal/noise ratio. You have well-defined and well-communicated SLOs (measured from the customer's perspective) which you always hit.
+    4. Feature development on your systems is effortless. Technical debt does not consume a large percentage of your team's time nor does it significantly slow down feature development. Engineers can't imagine an easier way to develop features in your system.
```
[the remark this answers is not in the document]
Comment (joannakrk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/gardener_tl_scorecard.md:30]: I think maybe it's ok for now... will think about it..

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/uber_tl_scorecard.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (joannakrk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/uber_tl_scorecard.md:25]: I think I'd just add a pragmatism, as the ability to balance business needs with the inner desire for ultimately beautful generic designs ;)

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (joannakrk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/warrior_tl_scorecard.md:29]: You optimize for low-latency process.
or
You optimize for quick turn-around at each step of the process?

**Review by joannakrk — APPROVED** [review]
I hope you can see comments...

File: recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/gardener_tl_scorecard.md [hunk 18]
```diff
@@ -0,0 +1,76 @@
...
+3. **_Team: you build and run a high-performing team and continually improve it_**
+    1. You realize it's not about you anymore. You put the team first in everything from giving recognition to the sexy projects to the promotion opportunities.
+    2. You are an amazing teacher. You effortlessly tie back implementation details to technologies, concepts, and entire thought domains. From 1:1 coaching to blog posts, tech-talks, or conferences, you scale your teaching appropriately. 
+    3. Your team runs itself. You are not a SPOF - multiple people can step in and take your role, at least temporarily. Every engineer has an equal voice and is comfortable using it. 
+    4. You are a talent magnet. People seek you out to work with you and your team. 
+    5. You are an effective partner to your EM for building a [Level 4 Tribe](https://en.wikipedia.org/wiki/Tribal_Leadership): the team has fun, loves what they do, and may be friends forever. 
+4. **_Execution/Process: you keep the team executing at peak-performance with minimal (but not too little!) process_**
+    1. You are actively contributing code (>50% of your time) and a top producer.
```
[the remark this answers is not in the document]
Comment (joannakrk) [reply recipes/management/increasing_the_capacity_to_win/growing/expectations/scorecards/gardener_tl_scorecard.md]: good point. Somehow I assumed... ;)

**Pull request merged**